src/amdgpu.cpp

```diff
@@ -171,7 +172,14 @@ void amdgpu_get_metrics(){
 	}
 
 	amdgpu_common_metrics_m.lock();
-	gpu_info.load = amdgpu_common_metrics.gpu_load_percent;
+
+#ifdef HAVE_LIBDRM_SAMPLING
+	if (!do_libdrm_sampling) {
+#endif
+		gpu_info.load = amdgpu_common_metrics.gpu_load_percent;
+#ifdef HAVE_LIBDRM_SAMPLING
+	}
+#endif
```

Comment:
The value gets overwritten, so the above change should be needed. Right?

Reply:
It seems like I've made an incorrect assumption that it should get overwritten, but it actually doesn't. Will remove in the next commit